contribs/gmf/src/directives/editfeature.js

```diff
@@ -0,0 +1,446 @@
+  // (1) Launch request
+  this.editFeatureService_.deleteFeature(
+    this.layer.id,
+    this.feature
+  ).then(
+    this.handleDeleteFeature_.bind(this)
+  );
+
+  // (2) Reset selected feature
+  this.feature = null;
+  this.features.clear();
+
+};
+
+
+/**
+ * Called after an insert, update or delete request.
+ * @param {angular.$http.Response} resp Ajax response.
+ * @private
+ */
+gmf.EditfeatureController.prototype.handleEditFeature_ = function(resp) {
+  var features = new ol.format.GeoJSON().readFeatures(resp.data);
+  if (features.length) {
+    var feature = features[0];
+    var id = feature.getId();
+    this.feature.setId(id);
+  }
+};
+
+
+/**
+ * Called after an insert, update or delete request.
+ * @param {angular.$http.Response} resp Ajax response.
+ * @private
+ */
+gmf.EditfeatureController.prototype.handleDeleteFeature_ = function(resp) {
+  console.log('ok, deleted');
+};
+
+
+/**
+ * @param {Array.<ngeox.Attribute>} attributes Attributes.
+ * @export
+ */
+gmf.EditfeatureController.prototype.setAttributes_ = function(attributes) {
+  this.timeout_(function() {
+    // Set attributes
+    this.attributes = attributes;
+
+    // Get geom type from attributes and set
+    var geomAttr = ngeo.format.XSDAttribute.getGeometryAttribute(
+      this.attributes
+    );
+    if (geomAttr && geomAttr.geomType) {
+      this.geomType = geomAttr.geomType;
+    }
+  }.bind(this), 0);
+};
+
+
+/**
+ * @param {ol.CollectionEvent} evt Event.
+ * @private
+ */
+gmf.EditfeatureController.prototype.handleFeatureAdd_ = function(evt) {
+  this.timeout_(function() {
+    var feature = evt.element;
+    goog.asserts.assertInstanceof(feature, ol.Feature);
+    this.feature = feature;
+    this.createActive = false;
+    this.scope_.$apply();
+  }.bind(this), 0);
+};
+
+
+/**
+ * Activate or deactivate this directive.
+ * @param {boolean} active Whether to activate this directive or not.
+ * @private
+ */
+gmf.EditfeatureController.prototype.toggle_ = function(active) {
+
+  var createUid = ['create-', goog.getUid(this)].join('-');
+  //var otherUid = ['other-', goog.getUid(this)].join('-');
+  var toolMgr = this.ngeoToolActivateMgr_;
+
+  if (active) {
+    toolMgr.registerTool(createUid, this.createToolActivate, false);
+    toolMgr.registerTool(createUid, this.mapSelectToolActivate, true);
+
+    //toolMgr.registerTool(otherUid, this.createToolActivate, false);
+    //toolMgr.registerTool(otherUid, this.modifyToolActivate, true);
+
+    this.mapSelectActive = true;
+    //this.modify_.setActive(true);
+  } else {
+
+    toolMgr.unregisterTool(createUid, this.createToolActivate);
+    toolMgr.unregisterTool(createUid, this.mapSelectToolActivate);
+
+    //toolMgr.unregisterTool(otherUid, this.createToolActivate);
+    //toolMgr.unregisterTool(otherUid, this.modifyToolActivate);
+
+    this.createActive = false;
+    //this.modify_.setActive(false);
+    this.mapSelectActive = false;
+    this.feature = null;
+    this.features.clear();
+  }
```

Comment:
In the other 7 PRs, I use those lines, i.e. they get un-commented.  I'll leave this like this.